templates/Devel.dockerfile.j2

```diff
@@ -15,6 +15,35 @@ ARG FROM_IMAGE=gpuci/rapidsai
 
 FROM ${FROM_IMAGE}:${RAPIDS_VER}-cuda${CUDA_VER}-devel-${LINUX_VER}-py${PYTHON_VER}
 
+ARG CCACHE_VERSION=master
+
+{# Install ccache from source (use specific version in projects commit history) #}
+RUN git clone https://github.com/ccache/ccache.git /tmp/ccache && cd /tmp/ccache \
+ && git checkout -b rapids-compose-tmp b1fcfbca224b2af5b6499794edd8615dbc3dc7b5 \
+ && ./autogen.sh \
+ && ./configure --disable-man --with-libb2-from-internet --with-libzstd-from-internet\
+ && make install -j \
+ && cd / \
+ && rm -rf /tmp/ccache-${CCACHE_VERSION}* \
```

Comment:
Looks like from line 21 you're installing to `/tmp/ccache` and checking out a specific commit on line 22. Doesn't look like the ARG on line 18 is used and the remove on line 27 appears that it would be leaving the install files given that you're using `/tmp/ccache`

May want to drop line 18, and change line 27 to match `/tmp/ccache`